### DISCUSSION TITLE
Unload chechpoints (just unload. not in the RAM)

### DISCUSSION THREAD

**Discussion by Ratinod** [comfyanonymous/ComfyUI#871] [Q&A]
Is it possible?
If there is not enough RAM, then the paging file is used (slow). It's much faster to just read and load a new checkpoint or load another checkpoint without touching the swap file.
"sdxl-base -> sdxl-refiner" workflow is pain...
<sub><sub>yes yes, I know... just buy more RAM... and just buy a graphics card with 24gb+ VRAM...</sub></sub>

**Comment by MaddyAurora** [comment]
wondering the same, in colab sdxl eats more and more ram till the run fails and restart is needed every few generated images
can't find a custom mode to unload the ram either
